doc/user/load_save.md

```diff
@@ -19,13 +22,169 @@ saving data and objects are also available.
+|              |                                       |      | [`arma_ascii`](https://arma.sourceforge.net/docs.html#save_load_mat) |
+|              |                                       |      | format. |
+|--------------------------------------------------------------------------------------------------------------|
+| `ARMA_BIN` | `.FileFormat() = FileType::ArmaBinary` | enum |  (autodetect extension `.bin`): Armadillo's     |
+|            |                                        |      |  efficient binary matrix format                 |
+|            |                                        |      |  ([`arma_binary`](https://arma.sourceforge.net/docs.html#save_load_mat)). |
+|--------------------------------------------------------------------------------------------------------------|
+| `RAW_ASCII` | `.FileFormat() = FileType::RawASCII` | enum | (autodetect extensions `.csv`, `.txt`):          |
+|             |                                      |      | space-separated values or tab-separated values (TSV) |
+|             |                                      |      | with no header.                                  |
+|--------------------------------------------------------------------------------------------------------------------|
+| `BIN_ASCII` | `.FileFormat() = FileType::RawBinary` | enum | (autodetect extension `.bin`): packed binary data     | 
+|             |                                       |      | with no header and no size information; data will be  |
+|             |                                       |      | loaded as a single column vector _(not recommended)_. |
+|--------------------------------------------------------------------------------------------------------------------|
+| `COORD_ASCII` | `.FileFormat() = FileType::` | enum | (autodetect extensions `.txt`, `.tsv`; must be loading a sparse |
+|               |                              |      | matrix type): coordinate list format for sparse data (see       |
+|               |                              |      | [`coord_ascii`](https://arma.sourceforge.net/docs.html#save_load_mat)). |
+|------------------------------------------------------------------------------------------------------------|
+| `AutoDetect_File` | `.FileFormat() = FileType::AutoDetect` | enum | auto-detects the format as one of the  |
+|                   |                                        |      | formats above using the extension of the |
+|                   |                                        |      | filename and inspecting the file contents.|
+|----------------------------------------------------------------------------------------------------------------|
+
+***Notes:***
+
+   - ASCII formats (`CSVASCII`, `RawASCII`, `ArmaASCII`) are human-readable but
+     large; to reduce dataset size, consider a binary format such as
+      `ArmaBinary` or `HDF5Binary`.
+   - Sparse data (`arma::sp_mat`, `arma::sp_fmat`, etc.) should be saved in a
```

Comment:
```suggestion
   - Sparse data (`arma::sp_mat`, `arma::sp_fmat`, etc.) must be saved in a
```
(Tiny little pedantic fix---there is no choice, these are the only supported formats for sparse data.)